modules/katamari/src/main/ts/ephox/katamari/api/Strings.ts

```diff
@@ -38,8 +39,13 @@ export const ensureTrailing = (str: string, suffix: string): string => {
   return endsWith(str, suffix) ? str : StrAppend.addToEnd(str, suffix);
 };
 
-export const contains = (str: string, substr: string): boolean => {
-  return str.indexOf(substr) !== -1;
+export const contains = (str: string, substr: string, start: number = 0, end?: number): boolean => {
```

Comment:
Sorry I missed this in my previous review, but this needs an `Improved` section added to the Katamari changelog. We also need to add `katamari@9.1.0` to `versions.txt` so we know it needs a minor bump.